go.mod

```diff
@@ -119,7 +119,7 @@ require (
 	gopkg.in/robfig/cron.v2 v2.0.0-20150107220207-be2e0b0deed5
 	gopkg.in/square/go-jose.v2 v2.6.0
 	gopkg.in/yaml.v3 v3.0.1
-	gorm.io/driver/postgres v1.3.8
+	gorm.io/driver/postgres v1.3.9
```

Comment:
@janisz do you know what's up with that? It looks like whenever one PR gets merged and as a consequence, dependabot opens up a new PR, that PR's diff will contain the changes from the just merged PR. Seems like a bug in dependabot?

Reply:
Or maybe it's not just a one PR but 2. When one is merged dependabot scan for new updates but if something else is merged in the meantime dependabot PR is stale. Maybe we should limit to 1 PR

Reply:
No, this is always with PRs that are being opened in response to dependabot quota becoming available again. The PR bumping gorm was merged 12:11 p.m., this one was created 12:13 p.m. I've observed this pattern before, and it was always the case that the PR has been opened just after the old one was merged.

Reply:
https://github.com/dependabot/dependabot-core/issues/4536#issuecomment-1250999825